website_sale_product_category_seo/views/templates.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data>
+
+    <template id="products_categories_cms" inherit_id="website_sale.products" name="Public Category CMS Block">
+        <xpath expr="//div[@id='products_grid']//*[1]" position="before">
+            <div t-if="category">
+                <div t-field="category.cms_top" id="category_cms_top"/>
```

Comment:
odoo core itself adds a placeholder already here. That this placeholder has not information which field or data object it is, is bad but should fix IMO in core.
Adding additional placeholder means that the page looks different in edit mode then in view mode and this is against the concept.
Odoo should implement a hover with information in the core, our placeholder is a bad workaround so this is the reason why i don't add it.